src/core/Cutter.h

```diff
@@ -34,6 +35,29 @@ class RizinTaskDialog;
 
 class RzCoreLocked;
 
+struct CUTTER_EXPORT CutterAddrRefs
```

Comment:
```suggestion
struct CUTTER_EXPORT AddrRefs
```

No need to prefix these structures with `Cutter`, unless there is a clash. For the `CutterJson` class however I do think it makes sense, to disambiguate it from `RzJson` and the Qt one.

Reply:
Changed with commit b9b778c4e3dac615e0e2944679eabeccec885354.